cptv-download.py

```diff
@@ -140,8 +141,16 @@ def _get_tags_descriptor_and_out_dir(self, r, filebase_name):
         return description, out_dir
 
     def _download(self, r, api, out_base):
-        dt = parse(r["recordingDateTime"])
-        file_base = dt.strftime("%Y%m%d-%H%M%S") + "-" + r["Device"]["devicename"]
+        dtstring = ""
+        if r["recordingDateTime"]:
```

Comment:
Changing this to a try block with a catch on parse exception will make it more full proof